opal/tools/wrappers/Makefile.am

```diff
@@ -62,7 +62,8 @@ endif # OPAL_INSTALL_BINARIES
 endif # OPAL_WANT_SCRIPT_WRAPPER_COMPILERS
 
 opal_wrapper_SOURCES = opal_wrapper.c
```

Comment:
we're fixing this as part of OPAL_CONFIG_THREADS refactor so it will not be in final PR
TODO: @hppritcha will remove as part of this rework.